app/components/Search/ResourcesRow.jsx

```diff
@@ -35,6 +35,55 @@ class ResourcesRow extends Component {
     });
   }
 
+  getOpenTime(schedule) {
+    let date = new Date();
+    let day = date.getDay();
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    let hour = date.getHours();
+    let closingOpeningTimes = {};
+    let currDay = days[day];
+    let result = { open: false };
+
+    schedule.forEach(item => {
+      closingOpeningTimes[item.day.replace(/,/g, '')] = {
+        open: item.opens_at,
+        close: item.closes_at
+      };
+    });
+
+    if(closingOpeningTimes[currDay] && hour < closingOpeningTimes[currDay].close) {
+      return {
+        open:true, 
+        time: timeToString(closingOpeningTimes[currDay].close)
+      };
+    } else {
+      let remainingDays = days.splice(day+1);
+      remainingDays.some(d => {
+        if(closingOpeningTimes[d]) {
+          result = {
+            open: false,
+            time: `${timeToString(closingOpeningTimes[d].open)} ${d}`
+          };
+          return true;
+        }
+      });
+
+      if(result.time) return result;
+
+      days.some(d => {
```

Comment:
I see--You want the next open day, so you want to start with the days after the current day. You could do it in one loop by splitting the array at the current day and then putting them back together with current day at the end. This is also fine, though.